servers/zms/src/main/java/com/yahoo/athenz/zms/store/ObjectStoreConnection.java

```diff
@@ -117,18 +117,22 @@
 
     // Policy commands
 
-    Policy getPolicy(String domainName, String policyName);
+    Policy getPolicy(String domainName, String policyName, String version);
     boolean insertPolicy(String domainName, Policy policy);
+    boolean insertPolicyVersion(String domainName, String version, Policy policy);
```

Comment:
why do we have this api instead of just using insertPolicy? 

see my comments in the implementation

Reply:
fixed